mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -1550,6 +1580,13 @@ func flattenSecondaryBootDisks(c []*container.SecondaryBootDisk) []map[string]in
 	return result
 }
 
+func flattenInsecureKubeletReadonlyPortEnabled(c *container.NodeKubeletConfig) bool {
```

Comment:
I don't think this is right, and wonder if a flatten function is even necessary, but struggled to get it working properly without one either.

Reply:
The flatten function probably isn't necessary for this field. What happens if you just use `c.InsecureKubeletReadonlyPortEnabled` instead of calling this function? It looks to me like the `c == nil` case is already covered.

Reply:
Thanks, yeah, updated! 👍